variational/permit.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+from eth_account import Account
+from eth_account.messages import encode_typed_data
+
+from .models import H160, Allowance
+from .client import Client
+
+
+class TransferPermitHelper(object):
+    def __init__(self, client: Client, private_key: str):
+        self.client = client
+        self.__private_key = private_key
+
+    def sign_and_submit(self, pool_address: H160, allowance: Allowance,
```

Comment:
i think we should simplify this to only take decimal or wei, not both

Reply:
How does this look?

````
def sign_and_submit(self, pool_address: H160, allowance: StrDecimal, seconds_until_expiry: Optional[int] = None)
def sign_and_submit_unlimited(self, pool_address: H160, seconds_until_expiry: Optional[int] = None)
```

And `generate_transfer_permit` would still take `Allowance` for flexibility.

Reply:
https://github.com/variational-research/variational-sdk-python/pull/7/commits/a33f6d6603a24cac77f4ec4c5c2ce305b1be1e23